_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,163 @@ required_by:
+| {% include anchor a="error.runtime.startup.noActiveAccount" %}                                                      | No AccountController could be found. You might have to login first.                                                                                                                                                                                                                           |
+| {% include anchor a="error.runtime.startup.noActiveConsumptionController" %}                                        | No ConsumptionController could be found. You might have to login first.                                                                                                                                                                                                                       |
+| {% include anchor a="error.runtime.startup.noActiveExpander" %}                                                     | No DataViewExpander could be found. You might have to login first.                                                                                                                                                                                                                            |
+| {% include anchor a="error.runtime.unknown" %}                                                                      | An unknown error occurred. Check the error message or the stack trace to learn more.                                                                                                                                                                                                          |
+| {% include anchor a="error.runtime.unknownType" %}                                                                  | The given '@type' could not be found during the deserialization.                                                                                                                                                                                                                              |
+| {% include anchor a="error.runtime.validation.invalidPayload" %}                                                    | The given combination of properties in the payload is not supported.                                                                                                                                                                                                                          |
+| {% include anchor a="error.runtime.validation.invalidPropertyValue" %}                                              | A property of the given payload is invalid.                                                                                                                                                                                                                                                   |
+
+# Error Codes regarding the transport ("error.transport.")
```

Comment:
Please add a link to the github repo. Also make sure to write "code" lowercase (same in other headlines).